ivy_tests/test_ivy/test_frontends/test_numpy/test_random/test_functions.py

```diff
@@ -136,6 +136,34 @@ def test_numpy_normal(
     )
 
 
+# shuffle
+@handle_frontend_test(
+    fn_tree="numpy.random.shuffle",
+    dtype_and_input=_get_dtype_and_matrix(),
+    test_with_out=st.just(False),
+)
+def test_numpy_shuffle(
+        dtype_and_input,
+        as_variable,
+        num_positional_args,
+        native_array,
+        frontend,
+        fn_tree,
+        on_device,
+):
+    input_dtype, x = dtype_and_input
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        as_variable_flags=as_variable,
+        with_out=True,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        frontend=frontend,
+        fn_tree=fn_tree,
+        test_values=False,
```

Comment:
given that we set `test_value=False`, we should make sure the return from test function is correct, for example we should check if the both are the same after sorting them out, checking the shape, dtype, etc.